controllers/flowcollector_controller.go

```diff
@@ -120,6 +126,7 @@ func (r *FlowCollectorReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	if r.consoleEnabled {
 		cpReconciler = consoleplugin.NewReconciler(clientHelper, ns, previousNamespace)
 	}
+	operatorsReconcilier := operators.NewReconcilier(ctx, clientHelper, ns)
```

Comment:
Nitpick:`Reconciler` instead of `Reconcilier` (for both variable and method names).